src/dune_lang/dune_lang.ml

```diff
@@ -162,6 +162,13 @@ module Ast = struct
     | Atom (_, s) -> Atom s
     | Quoted_string (_, s) -> Quoted_string s
     | List (_, l) -> List (List.map l ~f:remove_locs)
+
+  let rec add_loc sexp ~loc =
```

Comment:
Isn't this the same as `Dune_lang.add_loc`?

Reply:
It is! Don't know why I didn't notice. Removed it, thanks!